src/main/java/views/CureServiceViewController.java

```diff
@@ -113,7 +113,7 @@ public void onGenerateRegisterBtAction(ActionEvent event) {
 
         int pokemonLife = Integer.parseInt(lifeField.getText());
 
-        if (selectedPokemon.getLife() != pokemonLife
```

Comment:
Acho que nesse caso é pra ser `getLife` mesmo não? Pq se a vida do pokemon não é igual a vida que foi digitada pelo usuário, a vida do pokemon vai ser igual a vida digitada.

Reply:
Tinha algum bug relacionado a isso?

Reply:
se o treinador tiver mais de um pokemon e vc alterar a vida de um, depois mudar pra outro poke, e depois voltar pro primeiro poke, ele so gera o relatorio quando vc altera a vida denovo se o codigo for mantido assim

Reply:
Entendi, testei aqui e realmente tá bugado.